apps/website/src/content/docs/code.mdx

```diff
@@ -1,18 +1,21 @@
 ---
 title: Code
-description:
-propsPath: '@itwin/itwinui-react/esm/core/Typography/Code.d.ts'
+description: This component is used to define a specific piece of text as a short segment of code.
 thumbnail: Code
 ---
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='code' />
-
 <LiveExample src='Code.main.tsx' truncate={false}>
   <AllExamples.CodeMainExample client:load />
 </LiveExample>
 
+## Usage
```

Comment:
Since the usage is super simple, we do not need a separate section.

Suggestion:
1. Move the two sentences (lines 15-17) to be above the live example
2. Remove line 7 (frontmatter description)
3. Remove the "Usage" heading

Reply:
are you sure? you've done something completely different

Reply:
no, it gets displayed on this page: https://deploy-preview-1766--itwinui-previews.netlify.app/docs/components/

what you have right now is fine